Task 1/Task1.1.7/Array.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        public static void Sort(int[] a)
+        {
+            int min,index,i,j;
+            for (i = 0; i <=a.Length-1; i++)
+            {
+                index = i;
+                min = a[i];
+                for (j = i + 1; j < a.Length; j++)
+                {
+                    if (a[j] < min)
+                    {
+                        min = a[j];
+                        index = j;
+                    }
+                }
+                a[index] = a[i];
+                a[i] = min;
```

Comment:
```suggestion
            for (i = 0; i <=a.Length; i++)
            {
                for (j = i + 1; j < a.Length; j++)
                {
                    if (a[i] > a[j])
                    {
                        var temp = a[i];
                        a[i] = a[j];
                        a[j] = temp
                    }
                }
```

Можно проще. Плюс не нашел минимальное и максимальное значения (искать в отдельных методах).

Reply:
39 строка выводит минимальное и максимальное значения.

Reply:
А зачем менять сортировку посредством выбора на сортировку "пузырьком"?